blank-canvas-blocks/block-template-parts/navigation.html

```diff
@@ -1,4 +1,4 @@
-<!-- wp:navigation {"orientation":"horizontal","itemsJustification":"center"} -->
+<!-- wp:navigation {"orientation":"horizontal","itemsJustification":"center","isResponsive":true} -->
```

Comment:
Should we update Seedlet + Mayland's header template parts to include the responsive attribute?

Also, those template parts come with some embedded nav links which I think we should remove so the nav block appears in its empty state by default.

Reply:
Yes to both, I'll push a change.